package.json

```diff
@@ -8,11 +8,11 @@
     "format:write": "yarn prettier --write .",
     "lint": "echo 'Linting all packages' && wsrun --parallel --exclude-missing lint",
     "test": "echo 'Running tests' && wsrun --parallel --exclude-missing test",
-    "build": "echo 'Building all packages' && wsrun --stages --exclude-missing build",
+    "build-common": "echo 'Building all packages' && wsrun --stages --exclude-missing build-pre-app",
     "check": "run-p 'lint' 'format:check'",
     "prepare": "husky install",
     "precommit": "run-p 'format:staged'",
-    "watch:common": "nodemon --exec \"yarn build && yarn\"",
+    "watch:common": "wsrun --stages --exclude-missing watch",
```

Comment:
This looks like it would spawn 1 watch process per package, I think what @vyasworks was suggesting is a single watch at the root level

Reply:
@vyasworks Do we need to `watch` api/web? Or just common packages

Reply:
Nest/Next provide their own `watch`es right?

Reply:
Do we want watch for each package or just a common one? It would be easier and better if we had the ability to get local running with minimum commands. 

I am imagining something like this: 

# - DockerCompose Run - `Backing services` OR `Backing Services + API + Web` 
# - Start command in each module?
# - Yarn watch in the root

Reply:
@vyasworks `nodemon` was creating single watch for common packages. But that was not supper performance oriented .....

Reply:
@vyasworks and I will work on Dockerfile for creation of whole application execution with watch.